python/tvm/relay/transform/transform.py

```diff
@@ -666,7 +666,7 @@ def PartitionGraph():
     return _ffi_api.PartitionGraph()
 
 
-def AnnotateTarget(targets):
+def AnnotateTarget(targets, default_tuples=False):
```

Comment:
This API sounds quite ad-hoc to me. `Tuple` is just one of the nodes that can take the same target from its inputs/args. Incrementally adding parameters to handle different nodes would make the API bulky and code less readable, IMHO.

Reply:
Could you elaborate the list of non-call nodes that should have this feature along with the Tuple, please?

Reply:
I think what @zhiics means is all the overridden [Rewrite_]s of annotate_target except CallNode -- where Tuple is just one of them.

Reply:
For example, current implementation of `Let` in relay does not accept anything other than `Var` as its first parameter. Effectively, it cannot be annotated with anything but `"default"`. So for `Let` these changes would not make any difference.

Reply:
I think if skip_non_call_ops=True, we should not annotate Lets -- even with defaults. Thus, the behavior is not the same.